templates/web/base/admin/emergencymessage/edit.html

```diff
@@ -3,6 +3,14 @@
 
 [% INCLUDE status_message %]
 
+<style>
+.two-col {
+    display: grid;
+    grid-auto-flow: column;
+    grid-auto-columns: 1fr;
+}
+</style>
+
```

Comment:
Would be nice to have some headings to break the page up and make it clearer what page each message applies to.

```suggestion

<h2>Homepage</h2>
```